hack/install-deps.sh

```diff
@@ -27,14 +27,19 @@ function cmd() {
   exit $rc
 }
 
+# We don't need grunt and bower to be installed globally for the system,
+# so we can amend our path to look into the local node_modules for the
+# correct binaries.
+export PATH="${PATH}:$(pwd)/node_modules/bower/bin:$(pwd)/node_modules/grunt-cli/bin"
```

Comment:
> I think this only works correctly if you invoke the script as

Yep, thought this repo had the `cd` to the root dir as we did in Origin. Should fix that.

> Also, it's safer to move the export statements only within the if blocks (if the command isn't found, try to install it, then update the PATH so that it's available)

That would get the logic backwards -- those locations need to be in the `$PATH` for us to search for them in the first place